### DISCUSSION TITLE
Create an fully autonomous Script

### DISCUSSION THREAD

**Discussion by Pdzly** [ThioJoe/YT-Spammer-Purge#431] [Ideas]
I would like to just execute the script and it will search for some random channels which arent gotten "reported" or "deleted" the day before. So you could contribute to yt while afk this would be great too in a docker environment.

This would be great because i dont want to choose the channel / videos manually, yeah of course i have an api limit but when it hits then it should just stop :).

**Comment by cloudrootab** [comment]
Crowdsourcing randoms, e.g. trending videos and such. Would indeed be a good idea. Making it possible to automate reporting on a scheduled basis.

**Comment by MoralCode** [comment]
i think i suggested effectively the same thing in #465 without seeing this first

**Comment by ThioJoe** [comment]
Yea this is going into the "probably never going to happen" pile unfortunately. If I had an infinite amount of time sure, but there are plenty of other features I'd rather add first.
